lib/hbase/hbase-thrift/index.js

```diff
@@ -783,6 +784,47 @@ HbaseClient.prototype.getRows = function(options, callback) {
 };
 
 /**
+ * getAllRows
+ * get all rows in a table
+ */
+
+HbaseClient.prototype.getAllRows = function (options) {
+  var self = this;
+
+  return new Promise (function(resolve, reject) {
+    self.getScan({
+      table: options.table,
+      startRow: 0,
```

Comment:
I think there are actually chars before 0, I think it uses ASCII order http://www.asciitable.com/